src/common/arrstr.cpp

```diff
@@ -721,3 +730,212 @@ wxArrayString wxSplit(const wxString& str, const wxChar sep, const wxChar escape
 
     return ret;
 }
+
+
+namespace    // helpers needed by wxCmpGenericNatural()
+{
+    // Used for comparison of string parts
```

Comment:
Trivial, but we don't indent `namespace` contents to avoid indenting big blocks of text.

Reply:
Will fix, I did not change it from the original PR.